test/test_quantized.py

```diff
@@ -1323,6 +1322,129 @@ def test_qlinear_qnnpack(self, batch_size, input_channels, output_channels, use_
             np.testing.assert_equal(
                 Y_q_ref2.int_repr().numpy(), Y_q.int_repr().numpy())
 
+    @given(batch_size=st.integers(1, 3),
```

Comment:
why do we need separate tests for qnnpack? can't we just use the same set of tests as for fbgemm? (and iterate over available engines in the beginning)

same applies to linear tests above

Reply:
Ensuring numeric consistency of QNNPACK and FBGEMM is also a requirement for quantization aware training to work, so it will be good to check that both kernels pass the same tests. For QNNPACK we should also add tests that allow for full use of dynamic range of wts and activations

Reply:
Discussed offline. Will merge tests in a different PR.